tests/integration/test_velero.py

```diff
@@ -136,4 +146,16 @@ def test_integration_velero(module_instance: harness.Instance):
         backup_name,
     )
 
-    k8s_util.wait_for_deployment(module_instance, "nginx-deployment", "nginx-example")
+    k8s_util.wait_for_deployment(instance, "nginx-deployment", "nginx-example")
+
+
+def test_integration_velero_1_13_2(function_instance: harness.Instance):
```

Comment:
Let's make this a parameterized test to align with the other Rocks, wdyt?

Reply:
Done.